build-files/krona/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:jammy AS app
+
+ARG KRONA_VER="2.8.1"
+ARG KRAKENTOOLS_VER="d4a2fbe"
+
+LABEL base_image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="krona"
+LABEL software.version="${KRONA_VER}"
+LABEL description="Krona Tools is a set of scripts to create Krona charts from several Bioinformatics tools as well as from text and XML files."
+LABEL website="https://github.com/marbl/Krona"
+LABEL license="https://github.com/marbl/Krona/blob/master/KronaTools/LICENSE.txt"
+LABEL maintainer="Sage Wright"
+LABEL maintainer.email="sagemwright@gmail.com"
+
+# 'LABEL' instructions tag the image with metadata that might be important to the user
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    curl \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+    git \
+    make \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p ~/miniconda3 && \
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda3/miniconda.sh && \
+    bash ~/miniconda3/miniconda.sh -b -u -p ~/miniconda3 && \
+    rm ~/miniconda3/miniconda.sh
```

Comment:
I think the image might be smaller if the micromamba base is used instead of jammy + conda.

There's a template here: https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile_mamba

Reply:
I ended up just installing from root instead of using conda -- I think that ends up being even better for the size